src/main/java/Parser.java

```diff
@@ -0,0 +1,93 @@
+        tasks.add(new Task(s[1]));
+        String customeriseTopMes = uiDisplay.getTopTwoLine();
+        String taskMes = "   "+tasks.get(tasks.size()-1).toString();
+        String customeriseBottomMes = uiDisplay.getBottomTwoLine(tasks);
+        fileStorage.writeFile("todo",s[1],f);
+        return uiDisplay.parserOutputMess(customeriseTopMes,taskMes,customeriseBottomMes);
+    }
+
+    public String eventCommand(String command,ArrayList<Task> tasks,Ui uiDisplay,File f) throws Exception{
+        String [] s = command.split("event ");
+        String [] temp = s[1].split("/");
+        String taskName = temp[0];
+        String date = formatDate(temp[1].substring(3,temp[1].length()));
+        String customeriseTopMes = uiDisplay.getTopTwoLine();
+        tasks.add(new Deadline(taskName,date));
+        String taskMes = "     "+tasks.get(tasks.size()-1).toString();
+        String customeriseBottomMes = uiDisplay.getBottomTwoLine(tasks);
+        fileStorage.writeFile("deadline",taskName+date,f);
+        return uiDisplay.parserOutputMess(customeriseTopMes,taskMes,customeriseBottomMes);
+    }
+
+    public String deadlineCommand(String command,ArrayList<Task> tasks,Ui uiDisplay,File f) throws Exception{
+        String [] s = command.split("deadline ");
+        String[] temp = s[1].split("/");
+        String taskName = temp[0];
+        String date = formatDate(temp[1].substring(3,temp[1].length()));
+        String customeriseTopMes = uiDisplay.getTopTwoLine();
+        tasks.add(new Event(taskName, date));
+        String taskMes = "     " + tasks.get(tasks.size()-1).toString();
+        String customeriseBottomMes = uiDisplay.getBottomTwoLine(tasks);
+        fileStorage.writeFile("deadline",taskName+date,f);
+        return uiDisplay.parserOutputMess(customeriseTopMes,taskMes,customeriseBottomMes);
+    }
+
+    public String doneCommand(String command,ArrayList<Task> tasks,Ui uiDisplay){
+        if(tasks.size()>0){
+            String [] temp = command.split(" ");
+            int arrPos = Integer.parseInt(temp[1]);
+            tasks.get(arrPos-1).setDone();
+            String doneMesTop = uiDisplay.markAsDone();
+            String doneResult = "    ["+tasks.get(arrPos-1).getStatusIcon()+"] "+tasks.get(arrPos-1).getDescription();
+            String doneMesBot = "  -------------";
+            return uiDisplay.parserOutputMess(doneMesTop,doneResult,doneMesBot);
+        }else{
+            return "  Sorry there is no any task for you to do the command";
+        }
+
+    }
+
+    public void deleteCommand(String command,ArrayList<Task> tasks,Ui uiDisplay){
+        String [] temp = command.split(" ");
+        int arrPos = Integer.parseInt(temp[1]);
+        uiDisplay.removeTaskMes();
+        System.out.println("    "+tasks.get(arrPos-1).toString());
+        tasks.remove(arrPos-1);
+        uiDisplay.getBottomTwoLine(tasks);
+    }
+
+    public static String formatDate(String date) throws Exception{
```

Comment:
Might need to handle thrown Exceptions or consider the case of non-date-format deadlines/events. Users may not always want to enter deadlines/events in date format.